static/app/views/performance/browser/webVitals/components/webVitalDescription.tsx

```diff
@@ -44,30 +44,36 @@ const VITAL_DESCRIPTIONS: Partial<Record<WebVital, string>> = {
   ),
 };
 
-export function WebVitalDescription({score, value, webVital}: Props) {
+export function WebVitalDetailHeader({score, value, webVital}: Props) {
   const theme = useTheme();
+  return (
+    <Header>
+      <span>
+        <WebVitalName>{`${WEB_VITAL_FULL_NAME_MAP[webVital]} (P75)`}</WebVitalName>
```

Comment:
Needs to be translated? 🤔 Not sure though.

Reply:
The values in WEB_VITAL_FULL_NAME_MAP are already wrapped in translation, so we should be covered here!